docker/compose.sls

```diff
@@ -1,8 +1,6 @@
 {% from "docker/map.jinja" import compose with context %}
 
 compose-pip-dependencies:
-  pkg.installed:
-    - name: python-pip
   pip.installed:
     - name: docker-compose == {{ compose.version }}
     - require:
```

Comment:
This require will fail without the python-pip package installed. In addition the entire ID fails without pip.